CaseStudyProductListingTests/CaseStudyProductListingTests.swift

```diff
@@ -34,3 +35,5 @@ final class CaseStudyProductListingTests: XCTestCase {
     }
 
 }
+
```

Comment:
Why there is no unit tests? If not required delete this file or add appropriate tests